### PR TITLE
Add loading circle on health info UI

### PR DESCRIPTION
**Pull request by cesnietor**
fixes: https://github.com/minio/console/issues/558

Also hides the result of health info since it is only valuable when it is downloaded

Screen Shot 2021-01-19 at 3 53 48 PM

### CODE REVIEW

File: portal-ui/src/screens/Console/HealthInfo/HealthInfo.tsx [hunk 1]
```diff
@@ -54,6 +56,10 @@ const styles = (theme: Theme) =>
       border: "1px solid #EAEDEE",
       borderRadius: 4,
     },
+    loading: {
+      paddingTop: "8px",
+      paddingLeft: "40px",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/HealthInfo/HealthInfo.tsx:61]: Please remove unit as this is using pixels as default:

```suggestion
      paddingTop: 8,
      paddingLeft: 40,
```

File: portal-ui/src/screens/Console/HealthInfo/HealthInfo.tsx [hunk 2]
```diff
@@ -185,20 +190,13 @@ const HealthInfo = ({
...
-    return <pre>{str}</pre>;
-  };
   return (
     <React.Fragment>
       <PageHeader label="Diagnostic" />
+
       <Grid container>
         <Grid item xs={12} className={classes.container}>
-          <Grid container justify="flex-end" spacing={2}>
+          <Grid container justify="flex-start" spacing={2}>
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/HealthInfo/HealthInfo.tsx:199]: Can we move the justify property into a style?

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Comment by harshavardhana** [comment]
what happens when the page is refreshed? @cesnietor

**Comment by cesnietor** [comment]
> what happens when the page is refreshed? @cesnietor

progress is lost, you'll have to do it again since the health diagnostic is not stored anywhere.

File: portal-ui/src/screens/Console/HealthInfo/HealthInfo.tsx (hunk 2 above)
Comment (cesnietor) [reply portal-ui/src/screens/Console/HealthInfo/HealthInfo.tsx]: done

File: portal-ui/src/screens/Console/HealthInfo/HealthInfo.tsx (hunk 1 above)
Comment (cesnietor) [reply portal-ui/src/screens/Console/HealthInfo/HealthInfo.tsx]: done

**Review by bexsoft — DISMISSED** [review]
LGTM

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by Alevsk — APPROVED** [review]

**Pull request merged**